### ISSUE TITLE
App crash after hit any key

### ISSUE THREAD

**Issue by Serpic** [RiotGames/leaguedirector#33]
I tried to delete the settings (folder Documents\LeagueDirector), tried to connect another keyboard. The logs are empty.

Past topic: https://github.com/RiotGames/leaguedirector/issues/2

https://youtu.be/6pw-egk6Zr4

dmp-file: [LeagueDirector.exe.9156.zip](https://github.com/RiotGames/leaguedirector/files/3082235/LeagueDirector.exe.9156.zip)

[leaguedirector.log](https://github.com/RiotGames/leaguedirector/files/3080969/leaguedirector.log)

**Comment by Serpic** [comment]
There may be a problem in Windows (use 8.1), and maybe in the fact that I have a laptop.
It is strange that the program crashes only after pressing the buttons. All other functions of the program work.
For example, if you launch a replay of a game with a program, it will work, and if you switch (Alt+TAB) in program, it immediately crashes.

**Comment by cammm** [comment]
Thanks for the report and yes it sounds like 8.1 is the problem here. I am going to close this issue however, because it is a duplicate of this: #7
